includes/Core/Admin_Bar/Admin_Bar.php

```diff
@@ -272,11 +274,13 @@ private function is_amp_dev_mode() {
 	 * @since 1.0.0
 	 */
 	private function menu_markup() {
-		// Start buffer output.
+
 		ob_start();
```

Comment:
Is there any reason these functions use output buffering but ones like https://github.com/google/site-kit-wp/pull/1150/files#diff-893a4b5b38b9ce418fb9c1c2a99f0be9R208-R216 don't? Maybe there's a reason, but I don't understand what it is from the context here.